e2e_samples/dataset_versioning/docs/adrs/source_data_store.md

```diff
@@ -0,0 +1,19 @@
+# Source Data Store
+
+## Status
+
+Decided
+
+## Context
+
+In order to demonstrate data versioning, we need to pull from a data store. There is no existing _source_ data store, so we need to decide which technology we would like to use.
+
+## Decision
+
+We have decided on `Azure SQL` as our data store.
```

Comment:
Should we add what we compared with? 
We considered cosmos db 
Due to familiarity and usability for cosmosdb, we decided on sql.

Reply:
There are a few templates we can use. I chose this one, but happy to consider others.

https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md

Reply:
I guess this is more of personal preference. I like this one, [ard](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_madr.md).

I like this one because it does compare what we considered and later we can reason why we did "not" choose specific technology.

open to the table for more discussion tho.
cc: @devlace @DavidBurela

Reply:
Lets move the discussion outside of this PR. We can come pack and re-write it later.